databricks/koalas/tests/test_series.py

```diff
@@ -1775,6 +1775,25 @@ def test_mod(self):
         kdf = ks.from_pandas(pdf)
         self.assert_eq(kdf.a.mod(kdf.b), pdf.a.mod(pdf.b))
 
+    def test_mode(self):
+        pser = pd.Series([0, 0, 1, 1, 1, np.nan, np.nan, np.nan])
+        kser = ks.from_pandas(pser)
+        self.assert_eq(kser.mode(), pser.mode())
+        if LooseVersion(pd.__version__) >= LooseVersion("0.24"):
+            # The `dropna` argument is added in pandas 0.24.
+            self.assert_eq(
+                kser.mode(False).sort_values().values, pser.mode(False).sort_values().values
```

Comment:
Why do we use `.values`?

Reply:
The order of elements in the result series is different, for example
```
>>> pd.Series([0, 0, 1, 1, 1, np.nan, np.nan, np.nan])
0    0.0
1    0.0
2    1.0
3    1.0
4    1.0
5    NaN
6    NaN
7    NaN
dtype: float64
>>> pser = pd.Series([0, 0, 1, 1, 1, np.nan, np.nan, np.nan])
>>> kser = ks.from_pandas(pser)
>>> pser.mode(False).sort_values()
0    1.0
1    NaN
dtype: float64
>>> kser.mode(False).sort_values()
1    1.0
0    NaN
dtype: float64
```

Is this difference acceptable?

Reply:
I see, shall we `reset_index()` then?

```py
kser.mode(False).sort_values().reset_index()
```

Reply:
Btw, shall we use named argument to make the argument clear?

```py
kser.mode(dropna=False)
```

Reply:
Sounds good! Modified!